pynitrokey/cli/start.py

```diff
@@ -95,12 +95,7 @@ def rng(count, raw, quiet):
 
 @click.command()
 @click.argument("identity")
-@click.option(
-    "--force-restart",
-    is_flag=True,
-    help="Force pcscd and GnuPG services restart once finished",
-)
-def set_identity(identity, force_restart):
+def set_identity(identity):
```

Comment:
Did you maybe want to push this commit to #305?

Reply:
Ah, looks so indeed. Will look into that during the merge, good spot!